### ISSUE TITLE
Consulta 2 : Preguntar por el nombre de la tabla

### ISSUE THREAD

**Issue by enriquerussi** [IIC2413/Syllabus-2021-2#262]
Hola, quería saber si es valido preguntarle al usuario por el nombre de la tabla sobre la cual se quiere efectuar la acción, ya que esta no viene en los **kwargs. Por ejemplo, si se realiza un Distrito.create(**kwargs), dado que el nombre de la tabla no viene en esa información, se realizaría un input preguntándole al usuario sobre que tabla realizar el create (aún cuando el usuario sabe que es en la tabla distrito). Gracias.

**Comment by rjherrera** [comment]
no, no se le debe preguntar al usuario nada, no hay interacción con el usuario. El nombre de la tabla no viene en los kwargs, el nombre de la tabla lo sabes por el nombre de la clase.

**Comment by srodriguezp** [comment]
Hola! y existe alguna forma de ver que clase llamo a un metodo, esto ya que la clase model puede heredar varias veces y no se como identificar cual es la clase hija que esta llamando al metodo.

**Comment by rjherrera** [comment]
@srodriguezp te recomiendo mirar la issue #239
